airbyte-integrations/connectors/source-jira/unit_tests/test_components.py

```diff
@@ -112,10 +114,11 @@ def test_sprint_issues_substream_partition_router(fields_data, other_data, expec
     # Collecting results from stream_slices
     slices = list(router.stream_slices())
 
+    assert slices, "There should be at least one slice generated"
```

Comment:
This test would not pass because `fields_parent_stream_config.parent_key` was a mock and therefore `slices` would be and empty array. I've updated the test with the code on master and ported it here without any change